filebeat/module/postgresql/csv/_meta/fields.yml

```
@@ -0,0 +1,71 @@
+- name: log
```

Let's explicitly mark it as beta by now.
```suggestion
- name: csv
  release: beta
```